test-core/src/test/groovy/io/micronaut/starter/core/test/create/CreateMessagingSpec.groovy

```diff
@@ -38,7 +37,54 @@ class CreateMessagingSpec extends CommandSpec {
         where:
         [lang, buildTool, feature] << LanguageBuildCombinations.combinations(
                 beanContext.streamOfType(MessagingFeature.class)
+                        .filter({ f -> f.name != 'jms-sqs'} )
```

Comment:
I'll add a default instance of AmazonSQS in M2 and update the docs to explain that it must be overridden in client apps, and revert these test changes with that